src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Blur.Android.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Drawing;
+using Android.Content;
+using Android.OS;
+using Android.Views;
+using Android.Widget;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.UI;
+using Uno.UI.Controls;
+using Uno.UI.Xaml.Media;
+using View = Android.Views.View;
+
+namespace Windows.UI.Xaml.Media
+{
+	public partial class AcrylicBrush
+	{
+		private const float AndroidBlurRadius = 20;
+
+		private const float StyledBlurRadius = 64;
+
+		private static readonly Color DarkBlurOverlayColor = Color.FromArgb(128, 0, 0, 0);
+
+		//private static readonly Color LightBlurOverlayColor = Color.FromHex("#40FFFFFF");
+
+		//private static readonly Color ExtraLightBlurOverlayColor = Color.FromHex("#B0FFFFFF");
+
+		private static int blurAutoUpdateDelayMilliseconds = 100;
+		private static int blurProcessingDelayMilliseconds = 10;
+
+		private RealtimeBlurView _realtimeBlurView;
+
+		//private View _blurRootView;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)